docs/swagger.yaml

```diff
@@ -434,6 +434,103 @@ paths:
           $ref: '#/definitions/NotFound'
         500:
           $ref: '#/definitions/ServerError'
+  /submissions/{submissionId}/artifacts:
+    post:
+      summary: Create artifact for the given submission ID
+      description: |
+        Create artifact in S3
+        
+        **Authorization:** Create artifact follows the same authorization rules as Create submission
+      tags:
+        - Submissions
+      operationId: createArtifact
+      parameters:
+         - $ref: '#/parameters/submissionIdParam'
+         - in: body
+           name: typeId
+           required: true
+           schema:
+            type: string
+            example: 'c56a4180-65aa-42ec-a945-5fd21dec0503'
```

Comment:
We need 1 more parameter - for the artifact file itself

Reply:
keeping the swagger up to date and in sync is extremely important to us